WordPress/Classes/Extensions/NSString+Helpers.swift

```diff
@@ -3,9 +3,12 @@ import Foundation
 
 extension NSString
 {
-    /**
-     *  @details Splits the lines contained in the current string, and returns its unique values in a NSSet instance.
-     */
+    /// Returns the string's hostname. If there's any
+    public func hostname() -> String {
+        return NSURLComponents(string: self as String)?.host ?? String()
+    }
```

Comment:
I realize this was already there and just rewritten in swift but it's kind of terrible:

- `-[NSString hostname]` doesn't make much sense semantically
- If it did, this should return an optional

But most importantly, this is only used in a couple places, as `url.absoluteString.hostname()`, which would be equivalent to just calling `url.host`, which makes a lot more sense.